recipes/mono/mono-windows.recipe

```diff
@@ -25,9 +25,10 @@ class Recipe(recipe.Recipe):
         'bin/zlib1.dll',
         'etc/mono',
         'lib/mono/gac',
-        'lib/mono/2.0',
-        'lib/mono/4.0',
+        'lib/mono/2.0-api',
+        'lib/mono/4.0-api',
         'lib/mono/4.5',
```

Comment:
It was in the buildbot's folder `C:\Program Files\Mono\lib\mono`. Let me check if it's in a fresh install too.